### PR TITLE
Initial support for a monitoring issue

### PR DESCRIPTION
**Pull request by achamayou**

### CODE REVIEW

File: cimetrics/env.py [hunk 1]
```diff
@@ -146,7 +146,8 @@ def build_number(self) -> str:
 
     @property
     def is_pr(self) -> bool:
-        return "SYSTEM_PULLREQUEST_SOURCEBRANCH" in os.environ
+        return False
```
Comment (achamayou) [inline cimetrics/env.py]: Pretend this is master, sort of.

File: cimetrics/env.py [hunk 2]
```diff
@@ -170,8 +171,10 @@ def branch(self) -> Optional[str]:
 
     @property
     def pull_request_id(self) -> str:
-        assert self.is_pr
-        return os.environ["SYSTEM_PULLREQUEST_PULLREQUESTNUMBER"]
+        if self.is_pr:
+            return os.environ["SYSTEM_PULLREQUEST_PULLREQUESTNUMBER"]
+        else:
```
Comment (achamayou) [inline cimetrics/env.py:193]: Dubious, we probably want monitoring to happen only for a single, defined in config, branch.

Comment (achamayou) [reply cimetrics/env.py:193]: A more general implementation is probably to allow the config to specify an arbitrary set of branches, along with corresponding issues. That way release branches and so on can be supported.

**Comment by ghost** [comment]
monitoring_issue@[474 aka 20200413.2](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=474&view=results) vs master ewma over 9 builds from [445](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=445&view=results) to [471](https://dev.azure.com/jumaffre/metrics-devops/_build/results?buildId=471&view=results)
images

**Comment by achamayou** [comment]
Merging this now for testing in the cimetrics repo, and ideally in CCF as well. This needs refinement, but I think it's already basically useful and that can be done in later PRs.

**Pull request merged**